autopush/web/truss_validation.py

```diff
@@ -0,0 +1,118 @@
+import re
+import json
+import uuid
+
+from marshmallow import (
+    Schema,
+    fields,
+    pre_load,
+    validates_schema,
+)
+from cryptography.hazmat.primitives import constant_time
+from boto.dynamodb2.exceptions import ItemNotFound
+
+from autopush.exceptions import (
+    InvalidRequest,
+
+)
+from autopush.web.base import (
+    AUTH_SCHEMES,
+    PREF_SCHEME,
+)
+
+from autopush.utils import (
+    generate_hash,
+)
+
+
+class TrussSchema(Schema):
```

Comment:
Dropping, but Registration* was used by bridges as a support. Thus truss. Granted, we had some file name conflicts that have since gone away, so perfectly fine using registration again.